woodwork/serialize.py

```diff
@@ -1,225 +1,4 @@
-        if "engine" in csv_kwargs.keys():
-            del csv_kwargs["engine"]
-        if _is_koalas_dataframe(dataframe):
-            dataframe = dataframe.ww.copy()
-            columns = list(dataframe.select_dtypes("object").columns)
-            dataframe[columns] = dataframe[columns].astype(str)
-            csv_kwargs["compression"] = str(csv_kwargs["compression"])
-        dataframe.to_csv(file, **csv_kwargs)
-    elif format == "pickle":
-        # Dask and Koalas currently do not support to_pickle
-        if not isinstance(dataframe, pd.DataFrame):
-            msg = "DataFrame type not compatible with pickle serialization. Please serialize to another format."
-            raise ValueError(msg)
-        dataframe.to_pickle(file, **kwargs)
-    elif format in ["parquet", "arrow", "feather", "orc"]:
-        # Latlong columns in pandas and Dask DataFrames contain tuples, which raises
-        # an error in parquet and arrow/feather format.
-        latlong_columns = [
-            col_name
-            for col_name, col in dataframe.ww.columns.items()
-            if _get_ltype_class(col.logical_type) == ww.logical_types.LatLong
-        ]
-        if len(latlong_columns) > 0:
-            dataframe = dataframe.ww.copy()
-            dataframe[latlong_columns] = dataframe[latlong_columns].astype(str)
-        if format == "parquet":
-            dataframe.to_parquet(file, **kwargs)
-        elif format == "orc":
-            # Serialization to orc relies on pyarrow.Table.from_pandas which doesn't work with Dask
-            if _is_dask_dataframe(dataframe):
-                msg = "DataFrame type not compatible with orc serialization. Please serialize to another format."
-                raise ValueError(msg)
-            save_orc_file(dataframe, file)
-        else:
-            dataframe.to_feather(file, **kwargs)
-    else:
-        error = "must be one of the following formats: {}"
-        raise ValueError(error.format(", ".join(FORMATS)))
-    return {"location": location, "type": format, "params": kwargs}
-
-
-def _create_archive(tmpdir):
-    """When seralizing to an S3 URL, writes a tar archive."""
-    file_name = "ww-{date:%Y-%m-%d_%H%M%S}.tar".format(date=datetime.datetime.now())
-    file_path = os.path.join(tmpdir, file_name)
-    tar = tarfile.open(str(file_path), "w")
-    tar.add(
-        str(tmpdir) + "/woodwork_typing_info.json", arcname="/woodwork_typing_info.json"
-    )
-    tar.add(str(tmpdir) + "/data", arcname="/data")
-    tar.close()
-    return file_path
-
-
-def save_orc_file(dataframe, filepath):
-    from pyarrow import Table, orc
-
-    df = dataframe.copy()
-    for c in df:
-        if df[c].dtype.name == "category":
-            df[c] = df[c].astype("string")
-    pa_table = Table.from_pandas(df, preserve_index=False)
-    orc.write_table(pa_table, filepath)
+# This is included to prevent a breaking API change
+from woodwork.serializers.serializer_base import typing_info_to_dict
```

Comment:
Not sure - perhaps. I left this in for now, as Featuretools uses this function, so moving it would be a breaking change for Featuretools. However, if we are going to be limiting the min Woodwork version in Featuretools as a result of the pyspark pandas API changes anyway, we might be able to go ahead and remove this now.

Reply:
We'll likely increase the min Woodwork version in Featuretools, so we might as well remove this file.

Reply:
Sounds good to me